exercises/kindergarten-garden/kindergarten_garden_test.go

```diff
@@ -120,24 +128,29 @@ RVGCCGCV`, append([]string{}, test6names...), true, []lookup{
 func TestGarden(t *testing.T) {
 	for _, test := range tests {
 		g, err := NewGarden(test.diagram, test.children)
-		switch {
-		case err != nil:
-			if test.ok {
+		if !test.ok { // negative tests; expecting error
```

Comment:
By starting with the negative case, we end up with a double negative in the else. I have more trouble mentally thinking through the logic when there's a double negative. What do you think of swapping to start with `if test.ok`?